opennms-dao/pom.xml

```diff
@@ -114,7 +114,6 @@
     <dependency>
       <groupId>org.opennms.features.collection</groupId>
       <artifactId>org.opennms.features.collection.persistence.rrd</artifactId>
-      <scope>test</scope>
```

Comment:
It's just for accessing https://github.com/OpenNMS/opennms/pull/1245/files#diff-0a564fd806f532496281b87014032ac2R367. I can't do it the other way around because the dependency still exists for testing.